### DISCUSSION TITLE
Zigbee2Mqtt not starting anymore after changing the ssd controller - CC2531

### DISCUSSION THREAD

**Discussion by hajo62** [Koenkk/zigbee2mqtt#10224] [Devices]
I'm running my CC2531 since three years.
This week my ssd controller of my Pi crashed and I exchanged that (with another modell).
The system started up directly and nearly everything seems to be okay  -  except...

`Error: Failed to connect to the adapter (Error: SRSP - SYS - ping after 6000ms)`

The port hasn't changed. 
`lrwxrwxrwx 1 root root 13 Dec 18 11:47 usb-Texas_Instruments_TI_CC2531_USB_CDC___0X00124B00193648CA-if00 -> ../../ttyACM0`

After I have re-flashed the adapter (CC2531_router_2020_09_29.zip) , the LED is blinking red and the green isn't on and it isn't recognized by my Pi anymore. If I press the second button (not near the LED) the green light is on and the red one still blinking.

Than I reflashed with the diag-USB version. Same LED situation but the zigbee2mqtt now shows this error:
`Error: SRSP - SAPI - readConfiguration after 6000ms`

**Comment by HydrelioxGitHub** [comment]
I'm in the same situation here, does anyone have a solution for this ? I'm on a CC2652R

**Reply by hajo62** [reply]
In my case, it has been the dongle. I have bought a new one and it worked again without any changes.

**Comment by felamachado** [comment]
I have a similar situation.
Im migrating from SD to SSD.
The problem seems to be when SSD and CC2531 are both connected to RPI.
When i tested again with the SD Card all went fine. So now I'm again with my sd car plugged.

The solutions seems to be, test another SSD to USB adapter or buy another USB zigbee dongle (not CC2531).
